packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/Utilities/TypeSymbolExtensions.cs

```diff
@@ -163,18 +164,28 @@ private static CSharpType ConstructCSharpTypeFromSymbol(
         {
             var typeArg = namedTypeSymbol?.TypeArguments.FirstOrDefault();
             bool isValueType = typeSymbol.IsValueType;
-            bool isEnum = typeSymbol.TypeKind == TypeKind.Enum;
             bool isNullable = typeSymbol.NullableAnnotation == NullableAnnotation.Annotated;
```

Comment:
I was actually making some nullable related fixes in https://github.com/microsoft/typespec/pull/4712/files#diff-1de18efea7517e2bbe00376b0c79b1537d97944af8c4e9f727659db11089d0c2. I found that the NullableAnnotation is only specified if nullable references are not disabled for a project so I don't think we can rely on this.